tests/library/test_formats.py

```diff
@@ -312,6 +320,93 @@ def test_system_format(self):
 
         self.assertDictEqual(result, target)
 
+    # flake8: noqa: RUF001
+    def test_system_format_with_demos_different_target_prefixes(self):
+        settings = get_settings()
+        settings.allow_unverified_code = True
+
+        task, _ = fetch_artifact("tasks.qa.multiple_choice.with_topic")
+        task.input_fields["core_of_question"] = "str"
+
+        template = MultipleChoiceTemplate(
+            input_format="The following are multiple choice questions (with answers) about {topic}.\n{question}\nPotential Answers:\n{choices}\nAnswer:",
+            target_field="answer",
+            postprocessors=["processors.first_character"],
+            choices_separator="\n",
+            target_prefix="The {core_of_question} is: ",
+        )
+
+        card = TaskCard(
+            loader=LoadHF(path="cais/mmlu", name="elementary_mathematics"),
```

Comment:
Loading from HF is slow, so we should avoid it in unit test.  Instead you can use LoadFromDict to load a few examples in memory.